### DISCUSSION TITLE
is possible have this 2 software?? thanks

### DISCUSSION THREAD

**Discussion by faustf** [MichaIng/DietPi#6100] [Software Request]
hi is possible have this software in new version ? thanks

Ring language program
https://ring-lang.github.io/

and 
CCXT – CryptoCurrency eXchange Trading Library

github.com/ccxt/ccxt

thanks

**Comment by MichaIng** [comment]
Linking duplicate in forum: https://dietpi.com/forum/t/if-is-possible-have-this-2-software-many-thanks/15734

**Comment by MichaIng** [comment]
Thanks for your request. Could you open two dedicated requests here, filling this template: https://github.com/MichaIng/DietPi/issues/new?assignees=&labels=&template=software_request.md

You do not need to open it as issue, but copy&paste the Markdown template into the discussion.
